crates/notion-core/src/tool.rs

```diff
@@ -258,7 +272,7 @@ fn arg0(args: &mut ArgsOs) -> Fallible<OsString> {
 #[derive(Debug, Fail, NotionFail)]
 #[fail(display = "No {} version selected", tool)]
```

Comment:
There's an opportunity to make this error message better and more actionable, maybe something about how to install the missing user tool?